Glossario.json

```diff
@@ -1,322 +1,322 @@
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Ripiano": {
+		"description": "Ognuna delle superfici piane e orizzontali di uno scaffale.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Risk management": {
+		"description": "Pratica di individuazione, definizione, classificazione dei rischi a cui si può andare incontro durante il lavoro sul progetto, e delle relative pratiche di prevenzione.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Review": {
+		"description": "Revisione di documenti o codice. È un processo di peer review utilizzato per individuare eventuali problemi e migliorare la qualità del prodotto. È un'attività fondamentale per garantire uno standard qualitativo al prodotto.",
+		"acronyms": [],
+		"synonyms": ["Revisione"]
+	},
+	"Scaffale": {
+		"description": "Elemento presente all'interno del magazzino costituito da uno o più ripiani orizzontali che permettono lo stoccaggio della merce.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Scrum": {
+		"description": "Framework agile di gestione dello sviluppo di progetti complessi con l'obiettivo di consegnare il maggior valore business nel più breve tempo possibile.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Structured Query Language": {
+		"description": "Linguaggio standard per database basati sul modello relazionale, progettato per: creare e modificare schemi di database; inserire, modificare e gestire dati memorizzati; interrogare i dati memorizzati.",
+		"acronyms": ["SQL"],
+		"synonyms": []
+	},
+	"Scalable Vector Graphics": {
+		"description": "Formato di file di grafica vettoriale bidimensionale basato su XML sviluppato dal World Wide Web Consortium.",
+		"acronyms": ["SVG"],
+		"synonyms": []
+	},
+	"Three.js": {
+		"description": "Libreria JavaScript cross-browser utilizzata per creare e visualizzare grafica computerizzata 3D animata in un browser Web utilizzando WebGL.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"TypeScript": {
+		"description": "Linguaggio di programmazione fortemente tipizzato che si basa su JavaScript. Il codice TypeScript può infatti essere facilmente convertito in JavaScript, e funziona ovunque funzioni anche JavaScript.",
+		"acronyms": ["TS"],
+		"synonyms": []
+	},
+	"Typst": {
+		"description": "Linguaggio di markup progettato per la scrittura e formattazione di documenti.",
+		"acronyms": [],
+		"synonyms": []
+	},
+	"Unified Modeling Language": {
+		"description": "Linguaggio di modellazione visivo, destinato a fornire un modo standard per visualizzare la progettazione di un sistema.",
+		"acronyms": ["UML"],
+		"synonyms": []
+	},
+	"Use Case": {
+		"description": "Rappresentano le funzionalità del prodotto, che vengono rappresentate come interazioni tra utenti e sistema. Vengono spesso rappresentati tramite diagrammi realizzati con il linguaggio UML.",
```

Comment:
farei questo cambiamento:
"Rappresentano le funzionalità del prodotto, che vengono rappresentate come interazioni tra utenti e sistema." =>
"Caso (o situazione) in cui il sistema viene utilizzato per soddisfare uno o più requisiti degli utenti; cattura una parte di funzionalità che il sistema fornisce." (definizione presa dal libro Learning UML)